src/tilegarden/package.json

```diff
@@ -46,6 +46,7 @@
   },
   "dependencies": {
     "claudia-api-builder": "^4.1.0",
+    "lambda-warmer": "^1.0.0",
```

Comment:
The latest release is `1.1.0`.

Reply:
Weird.  I thought I had added that via `yarn add` just a few days ago.  Will fix.

Reply:
Fixed.  It was actually installing 1.1.0 already, since `^` only pins major version...